tests/common/plugins/allure_wrapper/allure_step_wrapper.py

```diff
@@ -0,0 +1,8 @@
+import logging
```

Comment:
well done, we should start using it for any new test, please inform the team

Reply:
> All in all looking good, Nana. Please review the minor comments. What is the test runtime?

It takes less than 30 mins to finish all the test cases